tools/device-listing/device-listing.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace Iot.Device.device_listing.Samples
```

Comment:
>device_listing [](start = 21, length = 14)

could you use something like Listing or DeviceListing? 
Also, the word Device became redundant here.

Reply:
forgot to change it